apps/service_define/priv/gettext/en/LC_MESSAGES/errors.po

```diff
@@ -0,0 +1,11 @@
+## `msgid`s in this file come from POT (.pot) files.
```

Comment:
Sucks we have so much boiler plate. 🤷‍♂

Reply:
Remove the comments at least. Don't think we can kill the file though